StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1012OpeningCurlyBracketsMustBeSpacedCorrectly.cs

```diff
@@ -111,6 +112,12 @@ private void HandleOpenBraceToken(SyntaxTreeAnalysisContext context, SyntaxToken
             followedBySpace = token.HasTrailingTrivia;
             lastInLine = followedBySpace && token.TrailingTrivia.Any(SyntaxKind.EndOfLineTrivia);
 
+            if (token.Parent as InterpolationSyntax != null)
```

Comment:
I wasnt sure if token.Parent could ever be null and the the behavior would be different. Could be possible in real (maybe broken) code?